docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
Other useful utilities from community hangout:
* Pretty printing of endpoints
* Service address/port lookup command(s)